data/changelog.txt

```diff
@@ -147,67 +154,82 @@ Monsters with larger eyes see farther than smaller variants.
 Fixes a segfault during initializing crafts.
 fixes segfault when attempting to siphon from vehicle with multiple tanks.
-Fix creatures blocking descend even if they can be displaced.
-Fix OnGetHit effects not triggering if player cannot block.
+Fixes creatures blocking descend even if they can be displaced.
+Fixes OnGetHit effects not triggering if player cannot block.
 Refactor monster spawning code to avoid crash.
 Reset game mode before loading normal saves.
 Minimize UI flickering on curses.
-Fix crash upon examining vehicles containing minireactor.
-Fix segfault on harvest.
-Fix loading of legacy characters without bodies.
-Fix explosions not affecting terrain.
+Fixes crash upon examining vehicles containing minireactor.
+Fixes segfault on harvest.
+Fixes loading of legacy characters without bodies.
+Fixes explosions not affecting terrain.
 Do not process held item twice.
-Fix crash when trying to reloading item inside advanced inventory
+Fixes crash when trying to reloading item inside advanced inventory
 Blobs are slimes. name them such.
-Fix freezing to death when wet.
-Fix weapon techniques not triggering.
+Fixes freezing to death when wet.
+Fixes weapon techniques not triggering.
 Prevent some obscure crashes by adding nullptr checks to unsafe pointer accesses.
 Vehicles: make sure going in reverse isn't more efficient than going forward.
-Fix doubled artifact effects on stats.
+Fixes doubled artifact effects on stats.
 Containers can't hold too large items contained in other containers.
 Prevent temperature being 0 after loading a save.
 Consume components in nested containers when crafting.
-Fix stuck joystick on Android.
+Fixes stuck joystick on Android.
 Adjust sunlight casting to illuminate exterior walls.
 Fixed limb blocking not triggering.
 Vehicles: make shock absorbers actually absorb shock damage.
-Fix NPC dying immediately on accepting inhaler mission
-Fix crash when attempting to butcher a corpse.
-Fix gear getting damaged when weather has no damaging effects.
+Fixes NPC dying immediately on accepting inhaler mission
+Fixes crash when attempting to butcher a corpse.
+Fixes gear getting damaged when weather has no damaging effects.
 Improve invalidation of max_populated_zlev cache, fixes lighting in some teleportation scenarios.
 Prevent dodging or blocking while insensible.
 Fixes apparent light asymmetry when the sun shines through a single-tile hole in the roof.
 Made installation time mandatory for gunmods.
 Makes tents and vehicles block sunlight.
-Fix scent map saved with invalid utf8 sequence.
+Fixes scent map saved with invalid utf8 sequence.
 Monsters no longer bash obstacles based on sound. This prevents groups of monsters from getting stuck in a loop of bashing based on the sound of other monsters bashing.
 Picking up an item stack fills up your pockets if it doesn't fit into one pocket.
 Wielding combines item stacks if possible.
 Robots can no longer be disabled through melee hits.
```

Comment:
```suggestion
Robots can no longer be disabled by a single melee hit.
```
Since hitting them enough that they fall apart does disable them (and more)...